gammapy/spectrum/energy.py

```diff
@@ -191,6 +191,41 @@ def lower_bounds(self):
         
         return self[:-1]
 
+    @property
+    def bands(self):
+        """Width of the energy bins
+        """
+        
+        upper = self.upper_bounds
+        lower = self.lower_bounds
+        return upper-lower
+
+
+    @classmethod
+    def from_lower_and_upper_bounds(cls, lower, upper, unit=None):
+        """EnergyBounds from lower and upper bounds (`~gammapy.spectrum.energy.EnergyBounds`). 
+
+        If no unit is given, it will be taken from upper
+        
+        Parameters
+        ----------
+        lower : `~astropy.units.Quantity`, float
```

Comment:
If you have such very similar arguments, you can combine them to save some typing for you and reading for the user:
```
lower, upper : `~astropy.units.Quantity`, float
    Lowest and highest energy bin
```